src/main/java/core/basesyntax/strategy/impl/SupplyService.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyService implements OperationHandler {
+    private FruitDao fruitDao = new FruitDaoImpl();

```

Comment:
```suggestion
    private final FruitDao fruitDao = new FruitDaoImpl();

```

It is a good practice to inject service objects using a constructor. Any class must not pe responsible for creating its dependencies